samba-integration-centos-ci-tests.sh

```diff
@@ -17,26 +17,31 @@ SCRIPT_GIT_BRANCH="centos-ci"
 SCRIPT_NAME="$(basename $0)"
 SCRIPT_PATH="$(realpath $0)"
 
-# enable additional sources for yum
-# (SCL repository for Vagrant, epel for ansible)
-yum -y install centos-release-scl epel-release
+# enable additional sources for yum:
+# - epel for ansible
+yum -y install epel-release
 
 # Install additional packages
 #
-# note: adding sclo-vagrant1-vagrant explicitly seems to fix
-#   issues where libvirt fails to bring up the vm with errors like this:
-#   "Call to virDomainCreateWithFlags failed: the CPU is incompatible with host
-#    CPU: Host CPU does not provide required features: svm" (or vmx)
```

Comment:
I hope this is no longer an issue with upstream vagrant as it is running successfully with heketi CI job.

Reply:
yes. hope this is consistently fixed now